src/hooks/useShowNotification.ts

```diff
@@ -0,0 +1,45 @@
+import { NotificationData, showNotification } from "@mantine/notifications";
+import { MessageKeys, NestedKeyOf, useTranslations } from "next-intl";
+import { useCallback } from "react";
+
+type NotificationType = "success" | "error";
+
+interface ShowNotificationProps extends NotificationData {
+  type: NotificationType;
+}
+
+const colorsMap: Record<NotificationType, string> = {
```

Comment:
I presume `Map` suffix is a little redundant since it may cause confusion with real `Map` structure in JS. How about just naming it `colors` or `notificationColors`? Same case for `titleKeysMap`